app/Policies/UserPolicy.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Policies;
+
+use App\Role;
+use App\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class UserPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether one user can change the Repair Directory role of another to a specific value.
+     *
+     * @param  \App\User  $user
+     * @param  \App\User  $user
+     * @param  int  $role
+     * @return mixed
+     */
+    public function canChangeRepairDirRole(User $perpetrator, User $victim, int $role)
+    {
+        # We have rules for whether you can change the Repair Directory role.  Code is structured for readability
+        # of these rules, rather than a single big if.
+        #
+        # Default to forbidden.
+        $ret = FALSE;
+        error_log("Can change role? {$perpetrator->repairdir_role()} . {$victim->id} vs {$perpetrator->id}" );
```

Comment:
If this is to be kept I would use Log::info from Illuminate\Support\Facades\Log (https://laravel.com/docs/5.6/logging)

Reply:
Sorry, yes, that will go.